server/src/controllers/authController.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const generateAccessToken = require('../utils/generateAccessToken');
+const { validationResult } = require('express-validator');
+const createUserService = require('../services/createUserService');
+const comparePasswordsService = require("../services/comparePasswordsService");
+const findUserByNameService = require("../services/findUserByNameService")
+
+class AuthController {
+  async registration(req, res) {
+    try {
+      const errors = validationResult(req);
+      if (!errors.isEmpty()) {
+        return res.status(400).json({ message: errors });
```

Comment:
return json must follow the form:
{
status: 'success' || 'failed',
message: 'message' || err.message
}